src/Api.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace ASC;
+
+class Api
+{
+    private $app;
+
+    public function __construct() {
```

Comment:
__[PHP_CodeSniffer]__ Opening brace should be on a new line ([link](https://sider.review/gh/repos/52858877/pulls/35#issue-3868045))

<sub>posted by Sider</sub>